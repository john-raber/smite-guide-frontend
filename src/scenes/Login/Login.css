label {
  float: left;
  color: rgb(255, 255, 255);
}

.container {
  margin-top: 3em;
}
