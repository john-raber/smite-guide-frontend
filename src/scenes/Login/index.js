import React from "react";
import { withRouter } from "react-router-dom";
import { Container, Row, Col, Jumbotron } from "reactstrap";

import "./Login.css";
import LoginForm from "./components/LoginForm/index";
import SignUpForm from "./components/SignUpForm/index";

const Login = () => {
  return (
    <Container>
      <Row>
        <Col md={{ size: 3 }}>
          <h3>Login</h3>
          <LoginForm />
        </Col>
        <Col md={{ size: 3 }}>
          <h3>Sign Up</h3>
          <SignUpForm />
        </Col>
      </Row>
    </Container>
  );
};

export default withRouter(Login);
