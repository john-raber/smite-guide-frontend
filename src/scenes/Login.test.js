import React from "react";
import { shallow } from "enzyme";

import { Login } from "./Login/index";

describe("Login", () => {
  it("renders correctly", () => {
    let wrapper = shallow(<Login />);

    expect(wrapper.find(".Login").length).toEqual(1);
  });

  it("matches its snapshot", () => {
    let wrapper = shallow(<Login />);

    expect(wrapper).toMatchSnapshot();
  });
});
