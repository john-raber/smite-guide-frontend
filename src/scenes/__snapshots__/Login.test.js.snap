// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login matches its snapshot 1`] = `
<Container
  className="Login"
  tag="div"
>
  <Row
    tag="div"
  >
    <Col
      md={
        Object {
          "size": 2,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    />
    <Col
      md={
        Object {
          "size": 3,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <h3>
        Login
      </h3>
      <withRouter(Connect(LoginForm)) />
    </Col>
    <Col
      md={
        Object {
          "size": 2,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    />
    <Col
      md={
        Object {
          "size": 3,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <h3>
        Sign Up
      </h3>
      <withRouter(Connect(SignUpForm)) />
    </Col>
    <Col
      md={
        Object {
          "size": 2,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    />
  </Row>
</Container>
`;
